src/main/resources/latest-spark-swagger-output.yaml

```diff
@@ -684,8 +737,25 @@ definitions:
         format: "date"
       id:
         type: "string"
-  AdminUser[]:
+  ResponseList:
     type: "object"
+    properties:
+      data:
+        type: "array"
+        items:
+          type: "string"
```

Comment:
This is going to be incorrect since at the very least the items will be an object. It might be necessary to have separate ResponseList types that include the correct reference to the subclass that they return.

Reply:
@binh-dam-ibigroup, do you have any thoughts on how to ammend this swagger def?

Reply:
The most thorough way to handle this would be to change the document generator class to create a `ResponseList_Subtype` object definition for each of the `ResponseList` responses we are publishing. We can also try creating a separate descendant of `ResponseBody` per @evansiroky's suggestion above and see how spark-swagger handles that.

Reply:
To follow up on my comment above, you could modify class `PublicApiDocsGenerator` to do the following:
- Scan all `ResponseList` return entries in the autogenerated swagger.
- For each instance, find the type of the elements in the list from one of the parent elements.
- Generate a custom return type definition that correspond to `ResponseList<type>`
- Replace `ResponseList` with the custom type generated above.

Reply:
Further discussion should be had in https://github.com/ibi-group/otp-middleware/issues/85